EXOSIMS/Observatory/TwoStarShades_mission.py

```diff
@@ -0,0 +1,567 @@
+from EXOSIMS.Observatory.SotoStarshade import SotoStarshade
+import numpy as np
+import astropy.units as u
+import astropy.constants as const
+import scipy.optimize as optimize
+
+
+class TwoStarShades_mission(SotoStarshade):
+    def __init__(
+        self,
+        scMass=[6000.0, 6000.0],
+        dryMass=[3400.0, 3400.0],
+        counter_1=0,
+        counter_2=0,
+        counter_3=0,
+        counter=0,
+        thrust=450,
+        **specs
+    ):
+        self.counter = counter
+        self.counter_1 = counter_1
+        self.counter_2 = counter_2
+        self.counter_3 = counter_3
+        SotoStarshade.__init__(self, **specs)
+
+        # occulter slew thrust (mN)
+        self.thrust = float(thrust) * u.mN
+
+        # occulters' initial wet mass (kg)
+        self.scMass = (np.array([scMass])) * u.kg
+        # print(self.scMass)
+        # occulters' dry mass(kg)
+        self.dryMass = np.array([dryMass]) * u.kg
+
+        # adding to outspec (Necessary for running ensemble)
+        self._outspec["scMass"] = scMass
+        self._outspec["dryMass"] = dryMass
+        # Acceleration
+        self.ao = self.thrust / self.scMass
+        # acceleration is an array
+        np.array([self.ao])
```

Comment:
This line of code doesn't do anything as the output is not assigned.

Reply:
fixed